projects/SelfTest/UsageTests/Message.tests.cpp

```diff
@@ -242,7 +242,7 @@ std::ostream& operator<<(std::ostream& out, helper_1436<T1, T2> const& helper) {
 }
 
 TEST_CASE("CAPTURE can deal with complex expressions involving commas", "[messages][capture]") {
-    CAPTURE(std::vector<int>{1, 2, 3}[0, 1, 2],
+    CAPTURE(std::vector<int>{1, 2, 3}[(0, 1, 2)],
```

Comment:
This should be instead handled by a localized warning suppression instead. The point of the test is to check that `CAPTURE` handles various weird expressions properly, including commas directly in subscript operator.